pkg/extensionAPI/version.go

```diff
@@ -6,7 +6,7 @@ import (
 	"golang.org/x/mod/semver"
 )
 
-const supportedApiVersion = "0.1.16"
+const supportedApiVersion = "0.2.0"
```

Comment:
In discussion we found out, that EM considers an interface to be "supported" if the major version is identical to that of the `supportedApiVersion`.
In this case the name or the value of `supportedApiVersion` in my eyes is a little bit misleading.
Proposal: 
```javascript
const supportedApiMajorVersion = "0"
```